test/int.jl

```diff
@@ -324,10 +324,16 @@ end
     end
 end
 
-@testset "issue #21092" begin
+@testset "Underscores in big_str" begin
     @test big"1_0_0_0" == BigInt(1000)
     @test_throws ArgumentError big"1_0_0_0_"
     @test_throws ArgumentError big"_1_0_0_0"
+
+    @test big"1_0.2_5" == BigFloat(10.25)
```

Comment:
Do these tests belong in `int.jl`? I guess it doesn't really matter...

Reply:
Fair point. It's a consequence of the `big"[float]"` code being in `base/int.jl`, which was true already before this PR.

Reply:
fair enough. Not worth fixing.